docs/changelog/1_0_a3.rst

```diff
@@ -0,0 +1,158 @@
+                               in default output mode (may fail if
+                               schema is updated after enabling
+                               option)
+    limit                      Set implicit LIMIT. Defaults to 100,
+                               specify 0 to disable
+    output-mode                Set output mode. One of: json, json-
+                               elements, default, tab-separated
+    verbose-errors             Print all errors with maximum
+                               verbosity
+
+
+EdgeQL
+======
+
+* Add ``+=`` and ``-=`` operations for use in :eql:stmt:`UPDATE`
+  (:eql:gh:`#165`).
+* Distinguish ``REQUIRED`` and ``OPTIONAL`` query parameters
+  (:eql:gh:`#1352`, :eql:gh:`#1355`)
+* Allow tuple arrays in schema definitions (:eql:gh:`73125882`).
+* Get rid of ``schema::CompoundType`` in favor of ``is_compound_type``
+  property (:eql:gh:`c30c1c4f`).
+* Add :eql:type:`anytype` and :eql:type:`anytuple` as proper abstract
+  types to the schema (:eql:gh:`affb65de`).
+* ``schema::Constraint`` no longer has ``args``, use ``params`` instead
+  (:eql:gh:`14cec6f1`).
+* Make ``std::Object`` the implicit base type for all user-defined
+  object types only.
+* Make ``std::BaseObject`` the root type for all object types both
+  user-defined and system object types.
+* Accept ``1e100n`` format as a valid ``bigint`` constant rather than
+  ``decimal`` (:eql:gh:`#1372`)
+* Make self-referencing (recursive) aliases forbidden.
+* Fix dependency tracking affecting index creation (:eql:gh:`#1181`).
+* Fix derivation of link targets in certain cases of multiple
+  inheritance (:eql:gh:`52c6b2d4`).
+* Fix handling of ad-hoc tuples (:eql:gh:`#1255`).
+* Fix incorrect implicit limit injection in subqueries in computables
+  (:eql:gh:`#1271`).
+* Computables cardinality must now be declared explicitly as
+  ``required``, ``single`` or ``multi``. The expression is validated
+  to be within the upper and lower limits implied by the declaration
+  (:eql:gh:`#1201`, :eql:gh:`#349`).
+* Prohibit database names longer than 63 characters (:eql:gh:`#1158`).
+* Prohibit duplicate elements in shapes (:eql:gh:`#1368`).
+
+
+GraphQL
+=======
+
+* Reflect "description" annotations into GraphQL descriptions
+  (:eql:gh:`#1228`).
+* Change the auto-generated type name template to use "\_Type" suffix
+  (:eql:gh:`#1175`).
+* Fix incorrect reflection of enums (:eql:gh:`#1227`).
+* Fix deeply nested insert issue (:eql:gh:`#1243`).
+
+
+Standard Library
+================
+
+* Add ``sys::get_current_database()``
+* Add ``std::to_array()`` string-splitting function.
+
+
+Server
+======
+
+* Make it possible to use different EdgeDB servers connected to the
+  same Postgres cluster.
+* Protocol: implement ``JSON_ELEMENTS`` IO format for responses
+  (:eql:gh:`#1169`).
+
+
+Misc
+====
+
+* Add cheatsheet.
+* Add an interactive tutorial.
+* Strictly type-annotate the core schema module.
+* Clean-up and improve schema and introspection code.
```

Comment:
Please reference Elvis' PR.

Reply:
There's more than one PR that cleans up schema code (like making collections more well-behaved, or generalizing introspection). Collectively they make schema and introspection work better, but I'm not sure that pointing to any individual PR is really all that enlightening.

Reply:
Please reference this pr https://github.com/edgedb/edgedb/pull/1408

> I'm not sure that pointing to any individual PR is really all that enlightening.

I'm not then sure why we are referencing any commits/prs?